### ISSUE TITLE
bug: Ionic 5 Refresher with icon overlays content

### ISSUE THREAD

**Issue by Unkn0wn0x** [ionic-team/ionic#20377]
# Bug Report

**Ionic version:**

[x] **5.x**

**Current behavior:**
`<ion-refresher>` overlays the content after upgrading from Ionic 4 to Ionic 5(RC).

**Expected behavior:**
`<ion-refresher>` does not overlay the content and works like in Ionic 4.

**Steps to reproduce:**
See fiddle here: [https://jsfiddle.net/mwjbh0fa/1/](https://jsfiddle.net/mwjbh0fa/1/)
Video: [https://streamable.com/ggge3](https://streamable.com/ggge3)

**Related code:**

I guess the behavior results in the changes of [this change](https://github.com/ionic-team/ionic/commit/fd55427991e94488d86971aaa10acb13d7fa1c23).

As far as I could debug it, the `<main>` element in the `shadow-root` is on the same `z-index` as the `<ion-refresher>` itself. In Ionic 4 the refresher was behind the content or in other words, the content overlayed the `<ion-refresher>`.

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.0.1 (/Users/[my-user]/.nvm/versions/node/v13.7.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.0.0-rc.2
   @angular-devkit/build-angular : 0.803.24
   @angular-devkit/schematics    : 8.3.24
   @angular/cli                  : 8.3.24
   @ionic/angular-toolkit        : 2.1.2

Cordova:

   Cordova CLI       : not installed
   Cordova Platforms : not available
   Cordova Plugins   : not available

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   Android SDK Tools : 26.1.1 (/Users/[my-user]/Library/Android/sdk/)
   ios-sim           : 8.0.2
   NodeJS            : v13.7.0 (/Users/[my-user]/.nvm/versions/node/v13.7.0/bin/node)
   npm               : 6.13.7
   OS                : macOS Catalina
   Xcode             : Xcode 11.3.1 Build version 11C504```

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you give the following dev build a try and let me know if it resolves your issue? `5.0.0-dev.202002042013.bc7d822`

**Comment by Unkn0wn0x** [comment]
Thanks for your fast response here too!

Works like a charm, great job 👏
